nircam_simulator/scripts/catalog_seed_image.py

```diff
@@ -1637,6 +1637,10 @@ def makePointSourceImage(self, pointSources):
             if bstr == "0.0":
                 bstr = "0.00"
 
+            if self.params['simSignals']['psfbasename'].lower() == 'niriss':
```

Comment:
I think you may need to rebase this from `master`. I merged some changes on Thursday that solved the PSF filename issue, such that two digits are always kept from the sub-pixel position.

Reply:
Rebasing may also fix the conflicts below.